src/js/action-types.js

```diff
@@ -16,11 +16,35 @@ export const SET_UPLOAD_DATA_TYPES = 'SET_UPLOAD_DATA_TYPES'
 export const SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME = 'SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME'
 export const SET_VISUALIZATION = 'SET_VISUALIZATION'
 
+export const DASHBOARD_CREATE_VISIBILITY = 'DASHBOARD_CREATE_VISIBILITY'
+export const DASHBOARD_CREATE_SUBTITLE_FORM = 'DASHBOARD_CREATE_SUBTITLE_FORM'
+export const DASHBOARD_CREATE_TITLE_FORM = 'DASHBOARD_CREATE_TITLE_FORM'
+export const DASHBOARD_CREATE_SAVE = 'DASHBOARD_CREATE_SAVE'
+export const DASHBOARD_DELETE_VISIBILITY = 'DASHBOARD_DELETE_VISIBILITY'
+export const DASHBOARD_EDIT_POPULATE_FORM = 'DASHBOARD_EDIT_POPULATE_FORM'
+export const DASHBOARD_EDIT_SUBTITLE_FORM = 'DASHBOARD_EDIT_SUBTITLE_FORM'
+export const DASHBOARD_EDIT_TITLE_FORM = 'DASHBOARD_EDIT_TITLE_FORM'
+export const DASHBOARD_EDIT_VISIBILITY = 'DASHBOARD_EDIT_VISIBILITY'
+export const DASHBOARD_EDIT_SAVE = 'DASHBOARD_EDIT_SAVE'
+export const DASHBOARD_SELECT = 'DASHBOARD_SELECT'
+
 // ASYNC
 
 export const FETCH_ANALYTICS_FAILURE = 'FETCH_ANALYTICS_FAILURE'
 export const FETCH_ANALYTICS_REQUEST = 'FETCH_ANALYTICS_REQUEST'
 export const FETCH_ANALYTICS_SUCCESS = 'FETCH_ANALYTICS_SUCCESS'
+export const FETCH_DASHBOARD_CREATE_FAILURE = 'FETCH_DASHBOARD_CREATE_FAILURE'
+export const FETCH_DASHBOARD_CREATE_REQUEST = 'FETCH_DASHBOARD_CREATE_REQUEST'
+export const FETCH_DASHBOARD_CREATE_SUCCESS = 'FETCH_DASHBOARD_CREATE_SUCCESS'
+export const FETCH_DASHBOARD_DELETE_FAILURE = 'FETCH_DASHBOARD_DELETE_FAILURE'
+export const FETCH_DASHBOARD_DELETE_REQUEST = 'FETCH_DASHBOARD_DELETE_REQUEST'
+export const FETCH_DASHBOARD_DELETE_SUCCESS = 'FETCH_DASHBOARD_DELETE_SUCCESS'
+export const FETCH_DASHBOARD_EDIT_FAILURE = 'FETCH_DASHBOARD_EDIT_FAILURE'
+export const FETCH_DASHBOARD_EDIT_REQUEST = 'FETCH_DASHBOARD_EDIT_REQUEST'
+export const FETCH_DASHBOARD_EDIT_SUCCESS = 'FETCH_DASHBOARD_EDIT_SUCCESS'
+export const FETCH_DASHBOARDS_FAILURE = 'FETCH_DASHBOARDS_FAILURE'
+export const FETCH_DASHBOARDS_REQUEST = 'FETCH_DASHBOARDS_REQUEST'
+export const FETCH_DASHBOARDS_SUCCESS = 'FETCH_DASHBOARDS_SUCCESS'
```

Comment:
How do you get one dashboards info? These action names don't lead me to the right answer.

So far we have only had to get content, not create any. The action names so far represent that. However, since some of these create things, they will likely veer away from the `FETCH` naming scheme.